<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YouTube Album Downloader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_download">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_url">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>URL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_artist">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>49</y>
      <width>34</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Artist</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_album">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>78</y>
      <width>39</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Album</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_year">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>107</y>
      <width>27</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_url">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>20</y>
      <width>276</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_artist">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>49</y>
      <width>276</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_album">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>78</y>
      <width>276</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_year">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>107</y>
      <width>276</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_number">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Track Numbers After Download</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>438</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
